client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -420,6 +424,72 @@ export class DomainWarnings extends React.PureComponent {
 		);
 	};
 
+	newTransfersWrongNS = () => {
```

Comment:
Could you please add some track events (or whatever works best) for these actions? Would be great to know how effective these notices are (especially if it'd be possible to create a funnel that'd show how many people actually clicked through till they updated the NS to WPCOM's).